autopush/db.py

```diff
@@ -246,7 +246,7 @@ def register_channel(self, uaid, channel_id):
         return True
 
     @track_provisioned
-    def unregister_channel(self, uaid, channel_id):
+    def unregister_channel(self, uaid, channel_id, **kwargs):
```

Comment:
In websocket.py it's called directly, in endpoint.py it's called via a addCallback. kwargs does eat the return result of the previous callback, but it's safe to do that since the return is irrelevant in that context.